README.md

```diff
@@ -1,26 +1,26 @@
-# ai-evals-orb
-# AI evals orb [![CircleCI Build Status](https://circleci.com/gh/CircleCI-Public/ai-evals-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/CircleCI-Public/ai-evals-orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/circleci/ai-evals.svg)](https://circleci.com/orbs/registry/orb/circleci/ai-evals) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/circleci-public/ai-evals-orb/main/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
+# evals-orb
+# evals orb [![CircleCI Build Status](https://circleci.com/gh/CircleCI-Public/evals-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/CircleCI-Public/evals-orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/circleci/evals.svg)](https://circleci.com/orbs/registry/orb/circleci/evals) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/circleci-public/evals-orb/main/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
 
-This repository has the code for the the CircleCI [AI Evals Orb](https://github.com/CircleCI-Public/ai-evals-orb). Please note that this version is currently in beta. Explore, understand, and experiment, but proceed with caution when integrating it into practical applications. 🛠✨
+This repository has the code for the the CircleCI [Evals Orb](https://github.com/CircleCI-Public/evals-orb).
 
 ## Usage
 
 ### Setup
```

Comment:
Is there something we should say here about the Github token necessary for PRs to be posted on GitHub?

Reply:
done.